chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -40,11 +40,13 @@
     mac_list = []
     unless mnode["network"].nil? || mnode["network"]["interfaces"].nil?
       mnode["network"]["interfaces"].each do |net, net_data|
-        net_data.each do |field, field_data|
-          next if field != "addresses"
-          field_data.each do |addr, addr_data|
-            next if addr_data["family"] != "lladdr"
-            mac_list << addr unless mac_list.include? addr
+        if !net.start_with?("tap", "qvo", "qbr", "qvb", "ovs")
```

Comment:
Files for bridges like br-int, br-tunnel, br-fixed are retained as there is only one instance of these for each node. They dont get accumulated for each VM. 

brqXXXX gets added for each VM for networkingplugin=linuxbridge. 
Currently ongoing test on local machine. Will update the patch adding to the list.